src/components/Carousel/index.vue

```diff
@@ -41,20 +42,17 @@
       await this.fetchFeatured();
     },
     methods: {
-      ...mapActions(['fetchFeatured']),
+      ...mapActions(app_type, ['fetchFeatured']),
       appImageBanner(assets) {
         const banner = assets.filter((asset) => asset.type == 'IB');
-        return banner[0].url;
+        return banner[0] ? banner[0].url : null;
```

Comment:
If the function here could return `undefined`, the following code would also work like this:
```suggestion
        return banner[0]?.url;
```